index.js

```diff
@@ -12,21 +12,22 @@ const {
 const { getUsers } = require('./utils/storage');
 const { deleteUser } = require('./models/users');
 const {
-    ideasDonationSum,
-    ideasSumDifference,
-    createDonation,
-} = require('./businessRules/donations');
+    getTotalSumDonatedForIdea,
+    createDonationByUnregisteredUser,
+    createDonationByRegisteredUser,
+} = require('./businessRules/donations.js');
```

Comment:
Why do you need to add `.js` here?

Reply:
I was desparate yesterday, befor you called ) Fixed